config/shoryuken.yml

```diff
@@ -1,4 +1,4 @@
-<% env = ENV['RAILS_ENV'] || 'development' %>
+<% prefix = ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV'] || 'development' %>
```

Comment:
Per CMS [ca69546](https://github.com/PRX/cms.prx.org/pull/598/commits/ca69546806e8b897c27aa24ad98ce61aecb0940f) ... this can just reference the Rails configuration.

And this PR also needs to update `shoryuken.rb` (I actually already removed the `DEFAULT_JOB_QUEUE_NAME` ENV from Infrastructure).